src/context/FeatureFlagContext.tsx

```diff
@@ -0,0 +1,86 @@
+import { useRouter } from "next/router";
+import { ParsedUrlQuery } from "querystring";
+import React, { createContext, useContext, useEffect, useState } from "react";
+import { extractQueryParam } from "../util/LinkUtils";
+
+interface FeatureFlag {
```

Comment:
Any reason why `interface` is preferred here over `type` ?

Reply:
Originally used this to define props for `FeatureFlagProvider`, will change this to `type`. According to the [TypeScript Handbook](https://www.typescriptlang.org/docs/handbook/2/everyday-types.html#differences-between-type-aliases-and-interfaces), they are mostly interchangeable and based on personal preference.

Reply:
Great --  I'm happy to continue using `type` by default so that uses of `interface` signal that we specifically need those features, aka when we're expecting people to extend the interfaces.